docs/topics/monitoring.md

```diff
@@ -6,14 +6,14 @@ There are five main options to monitor your cluster:
 
 1. [Kubectl](#kubectl)
 1. [Azure Monitor for containers](#azure-monitor-for-containers)
-1. [Kubernetes dashboard](#kubernetes-dashboard)
+1. [Kubernetes Dashboard](#kubernetes-dashboard)
 1. [Monitoring extension](#monitoring-extension)
 1. [Grafana and Influx DB](#grafana-and-influx-db)
 1. [Heapster REST API](#heapster-rest-api)
 
 ## Intro to Heapster
 
-Monitoring your cluster in Kubernetes is powered by a component called [Heapster](https://github.com/kubernetes/Heapster/). Heapster is a pod that is responsible for aggregating monitoring data from across all the nodes and pods in your cluster. Heapster is necessary for viewing monitoring data in the Kubernetes dashboard as well as in Grafana. Heapster comes preinstalled on `aks-engine` deployments. To ensure that Heapster is set up in your cluster and is running:
```

Comment:
blech, what should we do about this defuct heapster documentation?

Reply:
We should revise it for sure. This whole monitoring doc needs TLC, but I figured I'd just make dashboard-specific changes in this PR.